packages/sync/src/useSync.ts

```diff
@@ -136,6 +136,11 @@ export function useSync(opts: UseSyncOptions & TLStoreSchemaOptions): RemoteTLSt
 				client.close()
 				socket.close()
 				return
+			} else if (val === 'error' && closeCode === TLCloseEventCode.NOT_AUTHORIZED) {
```

Comment:
same comment here on adding logic here in the future to handle asking the user to login, instead of just closing the connection